src/Symfony/Component/Security/Core/Authorization/DebugAccessDecisionManager.php

```diff
@@ -29,7 +29,7 @@ class DebugAccessDecisionManager implements AccessDecisionManagerInterface
     private $voters;
     private $decisionLog = array();
 
-    public function __construct(AccessDecisionManager $manager)
+    public function __construct(AccessDecisionManagerInterface $manager)
```

Comment:
and ``setVoters`` should check that the wrapped implementation has this method too